### PR TITLE
New version: RandomEngy.VidCoder version 7.10.0

### PR DESCRIPTION
**Pull request by KaranKad**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/46012)

### CODE REVIEW

**Review by OfficialEsco — DISMISSED** [review]

**Comment by zachcarp** [comment]
@KaranKad - I ran this and I do see it installed, and I see it in the Control Panel, but I don't have any executables (per the error). When looking at the official project page it looks like it should have an executable. Could you help clarify what this project is installing?

**Comment by KaranKad** [comment]
```
2022-02-15 00:19:58.8360 [INFO] - Program: Starting Squirrel Updater: --setup C:\Users\WDAGUtilityAccount\AppData\Local\Temp\WinGet\RandomEngy.VidCoder.7.10.0.exe -s
2022-02-15 00:19:59.2577 [INFO] - Program: Extracting bundled app data from 'C:\Users\WDAGUtilityAccount\AppData\Local\Temp\WinGet\RandomEngy.VidCoder.7.10.0.exe'.
2022-02-15 00:20:03.3949 [INFO] - User32SplashWindow: User prompt suppressed (updater in silent mode): 'VidCoder requires .NET 6.0 Desktop Runtime (x64) installed to continue, would you like to install it now?' -- Automatically answering Cancel.
2022-02-15 00:20:03.4297 [INFO] - Program: Finished Squirrel Updater
```
Due to missing dependency in sandbox it the application does not install.

**Comment by denelon** [comment]
@KaranKad I see you already added the dependency to the manifest. 🚀

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/r/RandomEngy/VidCoder/7.10.0/RandomEngy.VidCoder.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: -s
+InstallerSuccessCodes:
+- 1
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.dotnetRuntime.6-x64
```
Comment (stephengillie) [inline manifests/r/RandomEngy/VidCoder/7.10.0/RandomEngy.VidCoder.installer.yaml:18]:
```suggestion
  - PackageIdentifier: Microsoft.DotNet.Runtime.6
```

**Comment by stephengillie** [comment]
Hi @KaranKad,

With the dependency preinstalled, I've tested installing this package in a VM, and it didn't seem to install anything. I couldn't find any program-specific folders under `Program Files` nor `Program Files (x86)`, and checked around in AppData too. Where should this package install its files?

**Comment by stephengillie** [comment]
Closing with reason: No recent activity.

**Comment by KaranKad** [comment]
Sorry for late response, when I have dependency installed it does install for me at `%localappdata%\VidCoder.Stable` by default.

**Pull request closed without merging**